spec/unit/puppet_x/puppetlabs/scheduled_task/trigger_spec.rb

```diff
@@ -121,6 +121,23 @@
       expect(canonical).to eq(expected)
     end
 
+    [
+      {'schedule' => 'boot'},
+    ].each do |event_manifest|
+
+      describe 'when validating event based triggers' do
```

Comment:
I'm wondering if we can use another term than `event`... but I can't think of anything better.

My concerns:

* There's an `IEventTrigger` that means something very specifically (though we'll probably never implement it)
* Arguably you could say that a trigger set for a specific date is responding to a time-based "event"

Maybe just referring to them everywhere as `event based` is sufficient?

Reply:
I also tried to come up with a term for these kinds of triggers that avoided the word event, but come up empty for something genuinely better.The saving factor is that we can name each type of event we want to support in the `schedule` property with names like `boot` or `logon`, but on the backend call them `event` since their properties are so similar. Personally I think it would be cool to support `IEventTrigger` some day, and we could call that one `event`, but as you say, maybe never get to it.

I'm fine with event based, but given that this describe string already says `event based`, are you asking for a change to this string or just the one below?

Reply:
I think as long as we stick to `event based` for now, its fine.